src/screens/RegisterOptionsScreen.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { StyleSheet, Text, View, Alert } from 'react-native';
+import { t } from 'ttag';
+import { useSelector } from 'react-redux';
+import { get } from 'lodash';
+import HathorHeader from '../components/HathorHeader';
+import NewHathorButton from '../components/NewHathorButton';
+import { COLORS } from '../styles/themes';
+import { NANO_CONTRACT_FEATURE_TOGGLE, REOWN_FEATURE_TOGGLE } from '../constants';
+
+const RegisterOptionsScreen = ({ navigation }) => {
```

Comment:
question: Was this screen displayed in the video?

Reply:
Yes! It is this screen:

![image](https://github.com/user-attachments/assets/9848a06b-8844-4934-bc05-b24ef6637faa)